### PR TITLE
[SimpleCalc] Doc updates - 5.2

### PR DESCRIPTION
**Pull request by vatsashah45**
## Contribution checklist (*)
 - [x] I have read the Contributing Guidelines
 - [x] Commits have been made with meaningful commit messages
 - [ ] All automated tests have passed successfully

### CODE REVIEW

File: simple-calc/art/Dark/Wizard/4.Presentation-MVUX.png
Comment (agneszitte) [inline simple-calc/art/Dark/Wizard/4.Presentation-MVUX.png:1]: @vatsashah45 don't forget the light version of the images as well
https://github.com/unoplatform/workshops/tree/dev/vs/5.2-Docs/simple-calc/art/Light/Wizard

**Review by agneszitte — COMMENTED** [review]
No code update at all @vatsashah45 ? did you redo the entire tutorial following the details for all options94 variants + Figma version) just to make sure there are no additional updates needed, please?

**Comment by agneszitte** [comment]
Related issue: https://github.com/unoplatform/uno-private/issues/365

@vatsashah45 make sure to add proper details for the title + description of the PR with at least the related issue, please. This helps to have more context if we need to go back to the changes.

**Comment by erikvilima** [comment]
@jeromelaban do we need to keep the mention to GTK, now that we are "GTKLess"?

https://github.com/unoplatform/workshops/blob/7e694c9cedc8b8263b68bc86ed31f0832755acaa/simple-calc/modules/Getting%20Started/README.md?plain=1#L18

cc @agneszitte

**Comment by erikvilima** [comment]
We can remove the "(in the SimpleCalculator project)" part here.

https://github.com/unoplatform/workshops/blob/7e694c9cedc8b8263b68bc86ed31f0832755acaa/simple-calc/modules/Resources/XAML/Creating-the-Layout.md?plain=1#L7

--
✅

**Comment by erikvilima** [comment]
`ControlExtensions` now is under the `Uno.Themes` namespace so we need to change it to `ut:ControlExtensions ...`

https://github.com/unoplatform/workshops/blob/7e694c9cedc8b8263b68bc86ed31f0832755acaa/simple-calc/modules/Resources/XAML/Creating-the-Layout.md?plain=1#L27

https://github.com/unoplatform/workshops/blob/7e694c9cedc8b8263b68bc86ed31f0832755acaa/simple-calc/modules/Resources/XAML/Creating-the-Layout.md?plain=1#L91-L94

-- 
✅

**Comment by erikvilima** [comment]
@vatsashah45 can you please check if we need to add a new namespace for `ControlExtensions` on C# Markup?

https://github.com/unoplatform/workshops/blob/7e694c9cedc8b8263b68bc86ed31f0832755acaa/simple-calc/modules/Resources/CSharp/Creating-the-Layout.md?plain=1#L114-L120

**Comment by erikvilima** [comment]
I also see that the Uno Figma Plugin layout has changed, should we consider updating the screenshots @agneszitte ?

@vatsashah45 In any case, it would be nice for you to follow the steps in the figma section and see if those steps still make sense and are accurate.

**Comment by jeromelaban** [comment]
> @jeromelaban do we need to keep the mention to GTK, now that we are "GTKLess"?

No, we should be referencing "Skia Desktop" instead. Overall, getting started should point to https://aka.platform.uno/get-started.

File: simple-calc/modules/Getting Started/README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ uid: Workshop.SimpleCalc.GettingStarted
 # Getting Started
 
 Uno Platform provides a multi-platform solution for building native apps for iOS, Android, Windows, macOS, Web, as well as Linux. This module will walk you through the process of getting started with Uno Platform.
+For additional information, make sure to check out this [Getting Started documentation] (https://aka.platform.uno/get-started).
```
Comment (jeromelaban) [inline simple-calc/modules/Getting Started/README.md]:
```suggestion
For additional information, make sure to check out this [Getting Started documentation](https://aka.platform.uno/get-started).
```

**Review by jeromelaban — APPROVED** [review]

**Review by erikvilima — APPROVED** [review]

**Pull request merged**